Eastern-Research-Group/expert-query
# Feature/69 basic client UI

## Related Issues:
* [EQ-69](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-69)
* [EQ-71](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-71)

## Main Changes:
* Added state management for the query inputs that will be available on the interface.
* Created React components using the USWDS library.
* Added the capability of updating the page URL when inputs change. Input fields are populated by parameters in the initial URL.
* Verified the EPA template being used is current.
   * A custom `max-width` value was set on the header, which doesn't match the official template, but if that is desired then it can be added back.

## Steps To Test:
1. Navigate to [http://192.168.0.130:3000/attains#dataProfile=catchmentCorrespondence&fileFormat=json&state=AR&state=GA](http://192.168.0.130:3000/attains#dataProfile=catchmentCorrespondence&fileFormat=json&state=AR&state=GA) and confirm that the fields are filled properly. In the displayed query boxes at the bottom of the page, the `fileFormat` parameter should be included in the _Current Query_ box, but not in the _Catchment Correspondence API Query_ box.
2. Clear a state from the _State_ input and confirm the URL updates.
3. Use a gibberish string for the file format, such as [http://192.168.0.130:3000/attains#dataProfile=assessmentUnitsMonitoring&fileFormat=cursive](http://192.168.0.130:3000/attains#dataProfile=assessmentUnitsMonitoring&fileFormat=cursive), and confirm it is replaced with the default format CSV.
4. Check that other design choices adhere to the US web design standards.
